services/order.py

```diff
@@ -0,0 +1,29 @@
+from db.models import Order
+from db.models import User
+from db.models import Ticket
+from django.db import transaction
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(tickets: list, username: str, date: str = None) -> Order:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+        order.save()
+
+    for elem in tickets:
+        movie_session = MovieSession.objects.get(id=elem["movie_session"])
+        Ticket.objects.create(movie_session=movie_session,
+                              row=elem["row"],
+                              seat=elem["seat"],
+                              order=order)
+    return order
+
+
+def get_orders(username: str = None) -> QuerySet:
```

Comment:
The `get_orders` function's return type should be `QuerySet[Order]` to accurately reflect the type of objects being returned.